coffeecard/CoffeeCard.Library/Services/SmtpEmailSender.cs

```diff
@@ -0,0 +1,19 @@
+using System.Threading.Tasks;
+using CoffeeCard.Common.Configuration;
+using MailKit.Net.Smtp;
+using MimeKit;
+
+namespace CoffeeCard.Library.Services;
+
+public class SmtpEmailSender(SmtpSettings smtpSettings) : IEmailSender
+{
+    public async Task SendEmailAsync(MimeMessage mail)
+    {
+        mail.From.Add(new MailboxAddress("Cafe Analog", "smtp@analogio.dk"));
+
+        using var smtpClient = new SmtpClient();
+        await smtpClient.ConnectAsync(smtpSettings.Host, smtpSettings.Port);
```

Comment:
This will throw a SocketException if the client cannot connect, e.g. if you are not running a local SMTP client. How should we handle this? Is it a valid assumption that a developer should have a client running, or should we handle the error nicely / ignore it?